types/lib/project-manager.d.ts

```diff
@@ -0,0 +1,294 @@
+import SerializableClass from "abitbol-serializable";
+import type { Structure } from "./structure";
+
+interface WprjOptions {
```

Comment:
could be exported

Reply:
https://github.com/wanadev/obsidian-project/pull/55/commits/81b8ff3b918a06b9ccda53c966e92166ad6502f3